monri/src/main/java/com/monri/android/model/Card.java

```diff
@@ -101,6 +104,7 @@ public String paymentMethodType() {
     };
     public static final String[] PREFIXES_MAESTRO = {"56", "58", "67", "502", "503", "506", "639", "5018", "6020"};
     public static final String[] PREFIXES_UNIONPAY = {"62"};
+    public static final String[] PREFIXES_DINACARD = {"9891"};
```

Comment:
@adnanomerovic just to check - where did you get this DinaCard prefix?

Reply:
@jasminsuljic from test cards, I noticed that 4 first digits are the same for all banks and I didn't find prefix on their official site.

Reply:
@adnanomerovic check task description on Jira regarding DinaCard bin range.